data/aecore/.minerva/accounts.json

```diff
@@ -1,3 +1,327 @@
 {
-  "ak_1GPPzM3VDKCP5RNEbp2uBNtgGTHRNQmrNkeAKGp7wfPWKYQvM": 6098700000000000000000
```

Comment:
I see no tests amended hence no automated check that this file is picked up when using the correct network id. Would it be possible to add a smoke test that with correct network id the code picks up this file e.g. hardcoding in the test one of the actual keys of this file (with non-zero balance) being bumped up in balance.

Reply:
Looks good for the rest.

Reply:
I ran a test locally on my machine. Will prepare a test. Good point

Reply:
added in [1a904bf](https://github.com/aeternity/aeternity/pull/2197/commits/1a904bf37242615a8fb40fdf7b2ee71b35976023)